stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/reactions/internal/ReactionsAdapter.kt

```diff
@@ -36,8 +37,11 @@ internal class ReactionsAdapter(
         private lateinit var reactionItem: ReactionItem
 
         init {
+            val isLollipop = Build.VERSION.SDK_INT <= Build.VERSION_CODES.LOLLIPOP_MR1
+            val isProblemDensity = context.resources.displayMetrics.densityDpi == 560
+            val additionalWidth = if (isLollipop && isProblemDensity) 10 else 0
             binding.root.updateLayoutParams<ViewGroup.MarginLayoutParams> {
-                width = itemSize
+                width = itemSize + additionalWidth
```

Comment:
I'm not sure should we keep it in our codebase or list it as "known issues". Looks strange. If we want to include it, I'd like to add some comment like "Please, don't touch it. Workaround for API less or equal than 23"

Reply:
Looks really odd 😄 Especially that additional width is in Px and not in DpToPx